docs/content/concepts/webserver/ui.mdx

```diff
@@ -100,6 +100,29 @@ width={1911}
 height={1049}
 />
 
+### Catalog views (Dagster+ Pro)
```

Comment:
```suggestion
</TabItem>
<TabItem name="Catalog views (Dagster+)">

### Catalog views (Dagster+)
```

Let's move this into its own tab so it's easier to find, and also consistent with how other pages in this reference are structured.